chainerx_cc/chainerx/native/native_device/misc.cc

```diff
@@ -16,6 +16,56 @@ namespace chainerx {
 namespace native {
 namespace {
 
+class NativePowKernel : public PowKernel {
+public:
+    void Call(const Array& x1, const Array& x2, const Array& out) {
```

Comment:
Ops, missing `override` here.
```suggestion
    void Call(const Array& x1, const Array& x2, const Array& out) override {
```